src/gui/import_metadata.c

```diff
@@ -196,34 +209,63 @@ void dt_import_metadata_init(dt_import_metadata_t *metadata)
 
     if(op_params_size == total_len)
     {
-      gtk_list_store_append(model, &iter);
+      gtk_list_store_append(metadata->m_model, &iter);
       // column 0 give the name, the following ones the different metadata
-      gtk_list_store_set(model, &iter, 0, (char *)sqlite3_column_text(stmt, 0), -1);
+      gtk_list_store_set(metadata->m_model, &iter, 0, (char *)sqlite3_column_text(stmt, 0), -1);
       for(unsigned int i = 0; i < DT_METADATA_NUMBER; i++)
       {
-        gtk_list_store_set(model, &iter, i+1, metadata_param[i], -1);
+        gtk_list_store_set(metadata->m_model, &iter, i+1, metadata_param[i], -1);
       }
     }
   }
   sqlite3_finalize(stmt);
+}
+
+void dt_import_metadata_init(dt_import_metadata_t *metadata)
+{
+  // default metadata
+  GtkWidget *grid = gtk_grid_new();
+  metadata->grid = grid;
+  gtk_box_pack_start(GTK_BOX(metadata->box), grid, FALSE, FALSE, 0);
+  gtk_grid_set_column_spacing(GTK_GRID(grid), DT_PIXEL_APPLY_DPI(5));
+  gtk_widget_show_all(grid);
+  gtk_widget_set_no_show_all(grid, TRUE);
+
+  // presets from the metadata plugin
+  GType types[DT_METADATA_NUMBER + 1];
+  for(unsigned int i = 0; i < DT_METADATA_NUMBER + 1; i++)
+  {
+    types[i] = G_TYPE_STRING;
+  }
+
+  metadata->m_model = gtk_list_store_newv(DT_METADATA_NUMBER + 1, types);
+  _import_metadata_presets_update(metadata);
 
   // grid headers
   GtkWidget *label = gtk_label_new(_("preset"));
   gtk_widget_set_visible(label, TRUE);
   gtk_label_set_ellipsize(GTK_LABEL(label), PANGO_ELLIPSIZE_END);
   gtk_widget_set_halign(label, GTK_ALIGN_START);
-  gtk_grid_attach(GTK_GRID(grid), label, 0, 0, 1, 1);
-  gtk_widget_set_tooltip_text(GTK_WIDGET(label), _("metadata to be applied per default"));
+  GtkWidget *labelev = gtk_event_box_new();
+  gtk_widget_set_visible(labelev, TRUE);
+  gtk_widget_add_events(labelev, GDK_BUTTON_PRESS_MASK);
+  gtk_container_add(GTK_CONTAINER(labelev), label);
+  gtk_grid_attach(GTK_GRID(grid), labelev, 0, 0, 1, 1);
+  gtk_widget_set_tooltip_text(GTK_WIDGET(label), _("metadata to be applied per default"
+                                                   "\ndouble-click on a label to clear the corresponding entry"
```

Comment:
The same action on the metadata editor does nothing, I bet will have request to do the same for consistency.

Reply:
I was keeping that for another PR, but I can add this too.

Reply:
> The same action on the metadata editor does nothing, I bet will have request to do the same for consistency.

Yes, it does nothing. But should it reset the image(s) metadata ?

Reset button should reset the preferences of the module (visible, private metadata). Only.

The button clear (already in place) clears metadata values.
I'll add the double-click on the label to clear the corresponding metadata.

OK ?

Reply:
I've raised #7859 to discuss the reset button on lib modules - imo it's not really very consistent or obvious so perhaps the reset button action is better discussed there?